src/components/FloatingHelper/FloatingHelper.tsx

```diff
@@ -28,24 +38,43 @@ const FloatingHelperBO = withStylable<PopoverProps, {}>(
 );
 
 export const FloatingHelper: React.SFC<FloatingHelperProps> = props => {
-  const {children, content, ...rest} = props;
+  const {children, content, width, ...rest} = props;
+  const contentWidth = (typeof width) === 'number' ? `${width}px` : width;
```

Comment:
no need, react does this for you https://reactjs.org/docs/dom-elements.html#style